MAUI/Scheduler/reminders.md

```diff
@@ -0,0 +1,425 @@
+---
+layout: post
+title: Appointment Reminders in .NET MAUI Scheduler control | Syncfusion
+description: Learn here all about the Reminder support in Syncfusion .NET MAUI Scheduler (SfScheduler) appointments.
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Appointment reminder in .NET MAUI Scheduler (SfScheduler)
```

Comment:
Appointments reminders in .NET MAUI Scheduler (SfScheduler)

Reply:
Changed to Appointments reminders in .NET MAUI Scheduler (SfScheduler)